automation/infrastructure/terraform/kcs/install_kubeflow/variables.tf

```diff
@@ -49,3 +49,19 @@ variable "kubeflow_version" {
   description = "version of kubeflow"
   default     = "1.3.0"
 }
+
+variable "kubeflow_monitoring" {
```

Comment:
should we make this named as install_prometheus or enable something etc? simpler to understand as a verb/action

Reply:
we should make it boolean. assuming thats the purpose

Reply:
Boolean. Done